Doxyfile

```diff
@@ -44,7 +44,7 @@ PROJECT_NUMBER         = @VERSION@
 # for a project that appears at the top of each page and should give viewer a
 # quick idea about the purpose of the project. Keep the description short.
 
-PROJECT_BRIEF          = "@CPACK_PACKAGE_DESCRIPTION_SUMMARY@"
+PROJECT_BRIEF          = "@DOXYGEN_PROJECT_BRIEF@"
```

Comment:
Yes, merge. Equalizer can use the short one. For all other use common sense, and we'll see in the PR. Thanks.